### ISSUE TITLE
Hide specific label on dynamic plotting

### ISSUE THREAD

**Issue by Hamza-malikx** [reactchartjs/react-chartjs-2#1067]
### Would you like to work on this feature?

- [ ] Check this if you would like to implement a PR, we are more than happy to help you go through the process.

### What problem are you trying to solve?

Hi guys, So I am mapping the data from api to plot multiple graphs based on data but the issue i am facing is that, I have a value in 1st graph but is not available for second graph why the label is still showing in the second graph then.

![image](https://user-images.githubusercontent.com/83468742/185766084-18939afa-4141-41b0-8b49-efd7bc1e2b4e.png)

### Describe the solution you'd like

Basically, I want a solution where we hide a specific label if it doesn't exist in data. Any help would be appreciated

**Comment by Arantiryo** [comment]
Hi @Hamza-malikx 👋 

Could you please share what data format are you using? 

The first thing that comes to mind is to use `ticks` callback to hide labels that meet a certain condition. You can add this to`options`:
```typescript
scales: {
    x: {
      ticks: {
        callback: function (value, index, ticks) {
          return data.datasets[0].data[index] === 0
            ? ""
            : this.getLabelForValue(value);
        }
      }
    }
```

This will hide labels that have `0` value associated with them. Here's [a code sandbox](https://codesandbox.io/s/zen-lederberg-evl7xt?file=/App.tsx). 

Do you think that would  work for your use case if you changed the condition?
